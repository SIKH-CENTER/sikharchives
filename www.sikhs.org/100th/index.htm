<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Sikhism Religion of the Sikh People</title>
<link href="..\sikhs.css" rel="stylesheet" type="text/css">

</head>

<body>

<div class="container">
  <div class="header">
  <!-- end .header -->
  
    <div class="logo"><a href="..\index.htm"><img src="..\images\Sikhs_org1.png" width="150" height="51" border="0"></a></div>
    
</div>
  <div class="navband1">
     <div class="navbar1">
    <a href="..\topics.htm">Home</a> 
    <a href="..\summary.htm">Introduction</a> 
    <a href="..\topics1.htm">Origins & Development</a> 
    <a href="..\topics2.htm">Philosophy & Scriptures</a> 
    <a href="..\topics3.htm">Way of Life</a>
    <a href="..\topics4.htm">Resources</a>
    </div>
  </div>
  <div class="page">
<div class="page_main">
    <div class="page_top">
        <div class="page_bottom">
<!--page content start -->
            <img src="images\centennial1a.jpg" width="250" height="119" hspace="10" vspace="10" align="left">
            <p class="pagetitle1">Century of Struggle and Sucess<br>
            The Sikh Canadian Experience</p>

          
          
<div class="breadcrumbs"><a href="..\topics.htm">Home</a> &gt;<a href="..\topics3.htm">Way of Life</a> &gt; <a href="..\memories\index.html">Sikh Memories</a> &gt; Sikhs in Canada</div>
<div class="clr"></div>

          <div class="box_right1">
            <p><a href="http://www.sikhmuseum.com"><img src="..\images\banner_sm1.png" border="0"></a></p>
          </div>
          <img src="images\stamp1.jpg" width="273" height="400" align="left">
          <p><em>&quot;The traveler comes and goes along the highway. What did he bring when he came, and what will he take away when he goes?&quot; (Guru Nanak Dev, Ramkali, pg. 931) </em></p>
          <p>&nbsp;</p>
          <p>Guru Nanak asks what have you accomplished on the highway of life? As Sikhs celebrate over 100 years in Canada we can ask 'What did we accomplish?' What have we contributed to the Canadian mosaic and what have we gained from that experience? This presentation is not meant to be a comprehensive history of our last 100 years in Canada, but instead an journey celebrating the Sikh experience. A journey which looks at select events in the history of Sikh pioneers to get a better understanding and appreciation of a century in Canada as seen through their eyes, actions and words.          </p>
          <div class="clr"></div>
          <table border="0" align="center">
            <tr>
              <td align="center" valign="top" width="231"><img height="491" src="images\kesur1.jpg" width="231"><br>
                <b><font face="ARIAL">Captain Kesur Singh</font></b><br>
                One of the first Sikh soldiers who arrived in Canada in 1897 following Queen Victoria's Diamond Jubilee.</td>
              <td valign="top"><p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="part1.html">Part I
                The Struggle Begins</a></p>
<p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="part2.html">Part II The Family</a></p>
              <p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="part3.html">Part III Community Strength</a></p>
              <p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="part4.html">Part IV Sense of Identity</a></p>
              <p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="part5.html">Part V Wartime Contribution</a></p>
              <p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="part6.html">Part VI Today &amp; Tommorow</a></p>
              <p><img src="images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="bibliography.html">Bibliography</a></p></td>
              <td align="center" valign="top" width="183"><img height="491" src="images\baltej1.jpg" width="183"><br>
                <b><font face="ARIAL">Baltej Singh Dhillon</font></b><br>
              In 1990 Baltej became the first Sikh RCMP officer allowed to wear a turban. </td>
              <td>&nbsp;</td>
            </tr>
          </table>
         
        </div>
    </div>
 </div>
  </div>
<div class="footer">
<div class="footer_resize">
      <p class="leftt">Copyright © 2011<a href="..\index.htm">Sikhs.org</a>. All Rights Reserved<br>
        <a href="..\about.html">About</a> | <a href="..\contact.html">Contact</a></p>
        <p class="rightt">&nbsp;</p>
      <div class="clr"></div>
    </div>
    <!-- end .footer --></div>
  <!-- end .container --></div>
</body>
</html>
