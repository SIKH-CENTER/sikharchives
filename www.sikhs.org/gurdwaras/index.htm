<html>

<title>The Sikhism Home Page: Historical Gurdwaras of Punjab</title>

<body bgcolor="#FFFFFF">



<center>
  <table border="0" width="600">
    <tr> 
      <td align="left"><font face="Arial, Helvetica, sans-serif" size="3"><b>Historical 
        Gurdwaras of Punjab</b></font></td>
      <td align="right"><img src="rkhanda.gif" width="60" height="70"></td>
    </tr>
  </table>
  <br>
  <p><i><font face="Arial, Helvetica, sans-serif" size="2">"True is that place, 
    where the mind becomes pure. True is the one who abides in Truth."<br>
    Guru Amar Das, Raga Gauri, pg. 158</font></i> <br>
    <br>
    <br>
  <p> 
  <b><font face="Arial, Helvetica, sans-serif" size="3">Historical Gurdwaras of 
  Punjab Classified by Guru </font></b> 
  <table border="0" cellspacing="10">
    <tr>
      <td><font face="Arial, Helvetica, sans-serif" size="3"><b><a href="guru1.htm">Guru 
        Nanak Dev</a><br>
        <a href="guru2.htm">Guru Angad Dev</a><br>
        <a href="guru3.htm">Guru Amar Das</a><br>
        <a href="guru4.htm">Guru Ram Das</a><br>
        <a href="guru5.htm">Guru Arjan Dev</a><br>
        <a href="guru6.htm">Guru Hargobind</a></b></font></td>
      <td><font face="Arial, Helvetica, sans-serif" size="2"><img src="t_gurdw.jpg" border="3" align="right"></font></td>
      <td><font face="Arial, Helvetica, sans-serif" size="3"><b><a href="guru7.htm">Guru 
        Har Rai</a><br>
        <a href="guru8.htm">Guru Harkrishan</a><br>
        <a href="guru9.htm">Guru Tegh Bahadur</a><br>
        <a href="guru10.htm">Guru Gobind Singh</a><br>
        <a href="famous.htm">Famous Sikhs</a></b></font><br>
      </td>
    </tr>
  </table>
  <br>
  <table border="0" width="600">
    <tr> 
      <td> <font face="Arial, Helvetica, sans-serif" size="3"><b>Gurdwaras</b></font><font face="Arial, Helvetica, sans-serif" size="2"><br>
        <br>
        <img src="zafarnama.jpg" border="4" align="right" width="200" height="300">All 
        religions have temples where people can gather together to contemplate 
        on God and pray. The Sikh temple is called a Gurdwara. The word 'Gurdwara' 
        means 'Gateway to the Guru'. In Sikhism ones personal dedication to living 
        a good life is important but another important aspect of Sikhism is the 
        Sangat (congregation). Not only should one meditate on God on their individual 
        level but also on a corporate level. There are thousands of Gurdwaras 
        throughout Punjab and the rest of the world. They serve as community centers 
        for the Sikh's. There are no restrictions on who may enter a Gurdwara 
        for prayer. People of all religions are welcome to attend. Another common 
        feature of all Gurdwaras around the world is Langer, the free community 
        kitchen. Here food is served to all people who sit together to enjoy a 
        communal meal. It is a symbol of the Sikh belief in a non-sexist, non-racist 
        society where all people of all casts, religions are equal and can share 
        a common meal in the true spirit of unity. </font> 
        <p><font face="Arial, Helvetica, sans-serif" size="2"> Many of the Gurdwaras 
          in Punjab have a pool (sarovar) for bathing in. In Sikhism one can bathe 
          in these pools if they wish, but they should be pure inside in order 
          to accomplish anything. For the water may clean you on the outside, 
          but it cannot clean you on the inside if your heart is not pure. Unlike 
          some religions, pilgrimages is not a part of Sikhism. Sikhs may visit 
          any Gurdwara as they are all considered equal because in all the living 
          Guru, Sri Guru Granth Sahib (the Sikh Holy Scriptures), is installed 
          as the spiritual head of the Sikh religion. <br>
          <br>
          <b><font size="3">In the Footsteps of the Gurus: The Photographs<br>
          </font></b><br>
          <img src="sandeep1.jpg" align="right">Sikhism is almost 500 years old 
          and the landscape of Punjab is scattered with Gurdwaras commemorating 
          important events in the lives of the Sikh Gurus. In early 1995 I spent 
          two months in Punjab visiting, photographing and documenting over 65 
          historical Gurdwaras. It's one thing to read about a religion and it's 
          history, it's quite another to experience it. Visiting the Gurdwaras 
          of Punjab really made the religion come alive for me. Suddenly I felt 
          transported back in time, walking in the same place where the great 
          Sikh Gurus walked. The experience had a very profound effect on me, 
          both emotionally and spiritually.</font> 
        <p> <font face="Arial, Helvetica, sans-serif" size="2">I originally had 
          gathered this material to publish a book, but publishing costs are very 
          high and I think that this precious resource material should be made 
          available to all Sikhs and non-Sikhs around the world. There is currently 
          no resource like it, in print, or in any other media. This web site 
          is my individual effort to help all people realize just how rich a history 
          and tradition Sikhism has. Sikhs should feel proud as the guardians 
          of this precious religious heritage and make every effort to preserve 
          it.</font><font face="Arial, Helvetica, sans-serif" size="2"> </font> 
        </p></td>
    </tr>
  </table>
  <p><font face="Arial, Helvetica, sans-serif" size="3"><b><br>
    Free Gurdwaras Windows Wallpapers </b></font> 
  <center>
    <table>
      <tr> 
        <td><font face="Arial, Helvetica, sans-serif" size="2">Download Windows 
          Wallpaper<br>
          of The Golden Temple </font></td>
        <td><img src="gld_sm.jpg"></td>
        <td><img src="bullet_k.gif" align="top"> <a href="/gld800.bmp"><font face="Arial, Helvetica, sans-serif" size="2">800x600, 
          24bit color (1.4 MB)</font></a><br>
          <img src="bullet_k.gif" align="top"> <a href="/gld640.bmp"><font face="Arial, Helvetica, sans-serif" size="2">640x480, 
          256 colors (292 K)</font></a></td>
      </tr>
      <tr> 
        <td><font face="Arial, Helvetica, sans-serif" size="2">Download Windows 
          Wallpaper<br>
          of Goindwal Sahib, the first<br>
          great centre of Sikhism</font></td>
        <td><img src="wp_goind.gif" width="122" height="95"></td>
        <td><img src="bullet_k.gif" align="top"> <a href="/goind1024.bmp" target=""><font face="Arial, Helvetica, sans-serif" size="2">1024x768, 
          24bit color (2.4 MB)</font></a><br>
          <img src="bullet_k.gif" align="top"> <a href="/goind800.bmp" target=""><font size="2" face="Arial, Helvetica, sans-serif">800x600, 
          24bit color (1.4 MB)</font></a></td>
      </tr>
      <tr> 
        <td align="CENTER" colspan="3"><font size="2" face="Arial, Helvetica, sans-serif"><b><font color="#FF0000">To 
          Download right-click on links and 'Save Link As' in<br>
          Netscape or 'Save Target As' is Internet Explorer</font></b></font></td>
      </tr>
    </table>
  </center>
  <br>
  <table border="0">
    <tr> 
      <td><a href="..\index.htm" target="_top"><img src="nc_circle.gif" width="51" height="50" border="0"></a><a href="..\whatnew.htm" target="_top"><img src="nc_whatsnew.gif" width="72" height="50" border="0"></a><a href="..\message.htm" target="_top"><img src="nc_comments.gif" width="76" height="50" border="0"></a><a href="..\dad.htm" target="_top"><img src="nc_dedication.gif" width="73" height="50" border="0"></a><a href="..\topics.htm" target="_top"><img src="nc_categories.gif" width="72" height="50" border="0"></a><a href="..\sitemap.htm" target="_top"><img src="nc_sitemap.gif" width="65" height="50" border="0"></a></td>
    </tr>
  </table>
  <br>
  <p>&nbsp;</p>
  <p><img src="copyrigh.gif" width="243" height="70"><br>
    <font size="1" face="Arial, Helvetica, sans-serif">Note: No graphics or photographs 
    from The Sikhism Home Page<br>
    may be reproduced or used on another web site or for any other <br>
    means without express written permission from Sandeep Singh Brar. </font></p>
</center>

</body></html>

















































