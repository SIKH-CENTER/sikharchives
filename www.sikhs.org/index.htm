<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Sikhism Religion of the Sikh People</title>
<link href="sikhs.css" rel="stylesheet" type="text/css">

</head>

<body>

<div class="container">
  <div class="header">
  <!-- end .header -->
  
    <div class="logo"><a href="index.htm"><img src="images\Sikhs_org1.png" width="150" height="51" border="0"></a></div>
    
</div>
  <div class="navband1"></div>
  <div class="page">
<div class="page_main">
    <div class="page_top">
        <div class="page_bottom">
<p>
  <!--page content start -->
&nbsp;</p>
<table width="800" border="0" align="center" cellpadding="10">
  <tr>
    <td><a href="topics.htm"><img src="images\NishanHome1.jpg" width="501" height="625" border="0"></a></td>
    <td><p><span style="font-size: 36pt; color: #666;">S</span><span style="font-size: 14pt; color: #666;">ikhism</span><br>
        <span style="font-size: 16px">A progressive 
        religion well ahead of its time when it was founded over 500 years ago, 
        The Sikh religion today has a following of over 20 million people worldwide 
        and is ranked as the worlds 5th largest religion. Sikhism preaches a message 
        of Devotion and remembrance of God at all times, truthful living, equality 
        of mankind and denounces superstitions and blind rituals. Sikhism is open 
        to all through the teachings of its 10 Gurus enshrined in the Sikh Holy 
        Book and Living Guru, Sri Guru Granth Sahib.</span></p>
      <p style="text-align: right"><font size="2" face="Arial, Helvetica, sans-serif"><a href="topics.htm">Enter website</a></font></p></td>
  </tr>
</table>
<p>&nbsp; </p>
          <div class="clr"></div>

		</div>
    </div>
 </div>
  </div>
<div class="footer">
<div class="footer_resize">
      <p class="leftt">Copyright © 2011<a href="index.htm">Sikhs.org</a>. All Rights Reserved<br>
        <a href="about.html">About</a> | <a href="contact.html">Contact</a></p>
        <p class="rightt">&nbsp;</p>
      <div class="clr"></div>
    </div>
    <!-- end .footer --></div>
  <!-- end .container --></div>
</body>
</html>
