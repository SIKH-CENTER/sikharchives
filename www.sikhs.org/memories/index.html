<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Sikhism Religion of the Sikh People</title>
<link href="..\sikhs.css" rel="stylesheet" type="text/css">

</head>

<body>

<div class="container">
  <div class="header">
  <!-- end .header -->
  
    <div class="logo"><a href="..\index.htm"><img src="..\images\Sikhs_org1.png" width="150" height="51" border="0"></a></div>
    
</div>
  <div class="navband1">
     <div class="navbar1">
    <a href="..\topics.htm">Home</a> 
    <a href="..\summary.htm">Introduction</a> 
    <a href="..\topics1.htm">Origins & Development</a> 
    <a href="..\topics2.htm">Philosophy & Scriptures</a> 
    <a href="..\topics3.htm">Way of Life</a>
    <a href="..\topics4.htm">Resources</a>
    </div>
  </div>
  <div class="page">
<div class="page_main">
    <div class="page_top">
        <div class="page_bottom">
<!--page content start -->
            <p class="pagetitle1">Sikh Memories</p>

          
          
<div class="breadcrumbs"><a href="..\topics.htm">Home</a> &gt;<a href="..\topics3.htm">Way of Life</a> &gt; Sikh Memories</div>
<div class="clr"></div>

          <div class="box_right1">
            <p><a href="http://www.sikhmuseum.com"><img src="..\images\banner_sm1.png" border="0"></a></p>
          </div>
          
          <p><img src="images\categories1.gif" width="417" height="448" usemap="#categories" border="0"><br>
              <em>click on a era above to view photographs
              <map name="categories">
                <area shape="rect" coords="7,105,220,147" href="in1.html"><area shape="rect" coords="7,168,218,208" href="in2.html"><area shape="rect" coords="6,229,221,268" href="in3.html"><area shape="rect" coords="8,289,218,327" href="in4.html"><area shape="rect" coords="5,350,219,387" href="in5.html">
              </map> 
            </em></p>
<table width="700" align="center" cellpadding="10">
            <tr>
              <td align="center" valign="bottom"><a href="..\ww1\index.html"><img src="images\ban_ww1.gif" width="216" height="130" border="0"></a><br>
                <i>The Sikh contribution in France</i></td>
              <td align="center" valign="bottom"><a href="..\100th\index.html"><img border="0" height="119" src="images\centennial1a.jpg" width="250"></a><br>
              <i>Celebrating the Canadian Centennial.</i></td>
            </tr>
          </table>
        </div>
    </div>
 </div>
  </div>
<div class="footer">
<div class="footer_resize">
      <p class="leftt">Copyright © 2011<a href="..\index.htm">Sikhs.org</a>. All Rights Reserved<br>
        <a href="..\about.html">About</a> | <a href="..\contact.html">Contact</a></p>
        <p class="rightt">&nbsp;</p>
      <div class="clr"></div>
    </div>
    <!-- end .footer --></div>
  <!-- end .container --></div>
</body>
</html>
