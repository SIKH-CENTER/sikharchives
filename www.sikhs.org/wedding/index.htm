<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Sikhism Religion of the Sikh People</title>
<link href="..\sikhs.css" rel="stylesheet" type="text/css">

</head>

<body>

<div class="container">
  <div class="header">
  <!-- end .header -->
  
    <div class="logo"><a href="..\index.htm"><img src="..\images\Sikhs_org1.png" width="150" height="51" border="0"></a></div>
    
</div>
  <div class="navband1">
     <div class="navbar1">
    <a href="..\topics.htm">Home</a> 
    <a href="..\summary.htm">Introduction</a> 
    <a href="..\topics1.htm">Origins & Development</a> 
    <a href="..\topics2.htm">Philosophy & Scriptures</a> 
    <a href="..\topics3.htm">Way of Life</a>
    <a href="..\topics4.htm">Resources</a>
    </div>
  </div>
  <div class="page">
<div class="page_main">
    <div class="page_top">
        <div class="page_bottom">
<!--page content start -->
            <p class="pagetitle1">Sikh Wedding</p>

          
          
<div class="breadcrumbs"><a href="..\topics.htm">Home</a> &gt;<a href="..\topics3.htm">Way of Life</a> &gt; <a href="..\fest.htm">Ceremonies and Festivals</a> &gt; Sikh Wedding</div>
<div class="clr"></div>

          <div class="box_right1">
            <p><a href="http://www.sikhmuseum.com"><img src="..\images\banner_sm1.png" border="0"></a></p>
          </div>
          <p><img src="wed_title1.gif" width="589" height="125"></p>
          <p><font face="Arial, Helvetica, sans-serif" size="2"><i>&quot;They are not 
        said to be husband and wife, who merely sit together. Rather they alone 
        are called husband and wife, who have one soul in two bodies.&quot; (Guru Amar 
        Das, Pauri, pg. 788) </i></font></p>
          <p><font face="Arial, Helvetica, sans-serif" size="3"><b>The Story Unfolds</b></font><font face="Arial, Helvetica, sans-serif" size="2"><br>
            <br>
            Although there may be some minor differences in the regional customs 
            leading up to the religious ceremony and following the ceremony, there 
            is very little variance in the actual Anand Karaj ceremony in the Gurdwara. 
            The Sikh wedding presented here is a typical example of a traditional 
            Sikh wedding being performed in a North American context. You can either 
            proceed through it sequentially watching the events unfold or jump to 
            any particular section.</font>
          </p>
          <table border="0" cellspacing="5">
            <tr>
              <td rowspan="10" valign="top"><img src="wed46.jpg" width="148" height="244"></td>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed1.html">Preparations 
                at Home</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed2.html">Departure 
                of the Braat</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed3.html">Arrival 
                at the Gurdwara</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed4.html">Performing 
                Ardas</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed5.html">The 
                Milni</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed6.html">Entering 
                the Gurdwara</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed7.html">Reading 
                of the Lavans</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed8.html">Conclusion 
                of the Religious Function</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed9.html">Well 
                Wishes from All</a></td>
            </tr>
            <tr>
              <td><img src="..\images\bullet_k.gif" width="22" height="28" align="absmiddle"> <a href="wed10.html">Aftermath</a></td>
            </tr>
          </table>
          <p>&nbsp;</p>
          <p><font face="Arial, Helvetica, sans-serif" size="3"><b>Anand Karaj - 
          A Union of Two Souls </b></font> </p>
          <p><font face="Arial, Helvetica, sans-serif" size="2"><img src="wed45.jpg" width="243" height="200" align="right">Anand 
            Karaj is the prescribed form of Sikh marriage, the words literally translate 
            as 'Blissful Union&quot;. The Sikh marriage is a very special ceremony 
            in which two individuals are joined in a equal partnership. It is joyous 
            and festive event which is very family orientated and informal in it's 
            atmosphere. Sikh marriages are usually arranged with families acting 
            as little more than introduction services. The ultimate choice is always 
            left to the girl and boy. In some cases the boy and girl choose each 
            other first and then seek their parents consent and blessing.</font> </p>
          <p><font face="Arial, Helvetica, sans-serif" size="2">The Rehat Maryada 
            which is The Official Sikh Code of Conduct specifies that no thought 
            should be given to the perspective spouses caste, race or lineage. As 
            long as both the boy and girl profess the Sikh faith and no other faith 
            they may be joined in wedlock by the Anand Karaj ceremony. The Rehat 
            Maryada strictly forbids any sort of dowry arrangement as marriage is 
            not to be viewed as a business transaction. Sikhs are also discouraged 
            from consulting horoscopes or following any other superstitions pertaining 
            to determining a wedding date or time. The Anand Karaj ceremony can 
            be performed in any Gurdwara or home where Sri Guru Granth Sahib has 
            been respectfully installed. The religious ceremony cannot be performed 
            in a hotel or banquet hall. There are no restrictions as to what time 
            the ceremony should start or what time it should end although they are 
            usually performed in the morning with the religious ceremony taking 
            no more than a few hours.</font> </p>
          <p><font face="Arial, Helvetica, sans-serif" size="2">An engagement ceremony 
            called the kurmai is not required but is sometimes performed typically 
            one week before the wedding. It is usually conducted in the Gurdwara 
            or at the home of the boy. It involves Ardas (the common Sikh prayer), 
            Kirtan (hymns from Sri Guru Granth Sahib) and langer (common meal) if 
            performed in the Gurdwara. If performed at home, the brides family visit 
            the house of the groom for a short time. In both cases the groom is 
            usually presented him with a kara, kirpan or Indian sweets. They brides 
            family in turn are presented with a Indian suit and sweets for the girl.</font> </p>
          <p><font face="Arial, Helvetica, sans-serif" size="2">In the east marriages 
            are typically a three day affair which begins with the Braat (grooms 
            family and friends) setting off for the girls house in the evening. 
            They are received, entertained and hosted by the girls family and spend 
            the night at her house. The following day the ceremony takes place at 
            the local Gurdwara or at the girls home. Following the religious ceremony 
            festivities, song and dance continue the remainder of the day with the 
            groom and his family departing with the bride the following day. In 
            the west because of the constraints in time and difficulty in coordinating 
            so many people, wedding are usually a one or two day affair. Either 
            the wedding occurs in the morning followed by a dinner and dance banquet 
            in a hall or hotel at night, or the religious ceremony and banquet take 
            place on consecutive days. In rarer cases only the religious ceremony 
            takes place without any banquet.</font></p>
        </div>
    </div>
 </div>
  </div>
<div class="footer">
<div class="footer_resize">
      <p class="leftt">Copyright © 2011<a href="..\index.htm">Sikhs.org</a>. All Rights Reserved<br>
        <a href="..\about.html">About</a> | <a href="..\contact.html">Contact</a></p>
        <p class="rightt">&nbsp;</p>
      <div class="clr"></div>
    </div>
    <!-- end .footer --></div>
  <!-- end .container --></div>
</body>
</html>
