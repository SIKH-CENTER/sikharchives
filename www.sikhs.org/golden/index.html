<html>
<head>
<title>The Sikhism Home Page</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
</head>
<body bgcolor="#FFFFFF" link="#FFFF00" vlink="#FFFF00" alink="#FFFF00">
<center>
  <p><img src="vtour.gif" width="451" height="150"> </p>
  <table border="0" width="500">
    <tr> 
      <td valign="top"> 
        <p><img src="t_pano.gif" width="269" height="26"><br>
          <font face="Arial, Helvetica, sans-serif" size="2"><img src="drawing1.gif" width="61" height="42" align="left">View 
          striking panoramic images of the Golden Temple Complex which make you 
          feel that your are standing there right now. The latest Immersive Imaging 
          technology has been used to create these panoramic images. Netscape 
          3 or Internet Explorer 3 are minimum requirements to view these panoramics. 
          Just click on any 'P' on the map below and wait for the image to finish 
          downloading. When it appears, click on it and drag your mouse to the 
          left or right to change your view.</font><br>
          <font face="Arial, Helvetica, sans-serif" size="2">
          <center>
            <img src="plan1.gif" width="379" height="337" border="0" usemap="#gplan">
          </center>
          </font> </p>
        <p><img src="t_movie.gif" width="269" height="26"> <br>
          <font face="Arial, Helvetica, sans-serif" size="2"><b><font color="#FF0000"><img src="drawing1.gif" width="61" height="42" align="left">Harimandir 
          Sahib: A Spiritual Journey</font></b><br>
          An inspiring and uplifting audiovisual celebration of the experience 
          of visiting Sri Harimandir Sahib. This 7.5 minute movie requires the 
          RealNetworks Real Player 5.0 to view, if you do not have it a link to 
          their website is provided. The movie can be viewed as a streaming video 
          by clicking on the link below which best describes your Internet connection 
          or as a downloadable file. If while watching the streaming movie, it 
          does not play without interruptions, download the zip file of the movie 
          to your hard drive instead. This higher quality movie file (golden.rm) 
          can be extracted from the zip file and then played locally from your 
          hard drive using Real Player 5.0.</font><br>
        </p>
        <table border="0">
          <tr> 
            <td><a href="golden28.ram"><img src="d_28.gif" width="70" height="67" border="0"></a></td>
            <td><a href="golden56.ram"><img src="d_56.gif" width="70" height="67" border="0"></a></td>
            <td><a href="goldenisdn.ram"><img src="d_isdn.gif" width="70" height="67" border="0"></a></td>
            <td><a href="golden.zip"><img src="d_zipf.gif" width="70" height="67" border="0"></a></td>
            <td><font face="Arial, Helvetica, sans-serif" size="2"><a href="http://www.real.com/products/player/downloadrealplayer.html?wp=dl0498&src=download"><img src="rplayer.gif" width="88" height="32" hspace="40" border="0"></a></font></td>
          </tr>
        </table>
        <br>
        <table border="0" width="100%">
          <tr> 
            <td colspan="2"><img src="t_links.gif" width="269" height="26"> </td>
            <td rowspan="4" valign="bottom" align="right"><a href="gall9.html"><img src="gall9s.gif" width="90" height="61" border="3" vspace="5" hspace="5"></a><br>
              <a href="gall10.html"><img src="gall10s.gif" width="90" height="61" border="3" vspace="5" hspace="5"></a></td>
          </tr>
          <tr> 
            <td><a href="http://www.sikhs.org/gurdw_a.htm"><img src="bullet.gif" width="38" height="39" border="0"></a></td>
            <td><font face="Arial, Helvetica, sans-serif" size="2">The Golden 
              Temple<br>
              (Gurdwaras category, Way of Life section)</font></td>
          </tr>
          <tr> 
            <td><a href="..\art8.htm"><img src="bullet.gif" width="38" height="39" border="0"></a></td>
            <td><font face="Arial, Helvetica, sans-serif" size="2">A Day at Darbar 
              Sahib by Patwant Singh<br>
              (Essays category, Resources section)</font></td>
          </tr>
          <tr> 
            <td><a href="..\other.htm#screen"><img src="bullet.gif" width="38" height="39" border="0"></a></td>
            <td><font face="Arial, Helvetica, sans-serif" size="2">Golden Temple 
              Windows Wallpaper<br>
              (Other Resources category, Resources section)</font></td>
          </tr>
        </table>
        <p>&nbsp; </p>
      </td>
      <td valign="top" align="right"><img src="t_photos1.gif" width="100" height="26"><br>
        <font face="Arial, Helvetica, sans-serif" size="1">click on images</font><br>
        <a href="gall1.html"><img src="gall1s.gif" width="60" height="90" vspace="5" hspace="5" border="3"></a><br>
        <a href="gall2.html"><img src="gall2s.gif" width="60" height="90" vspace="5" hspace="5" border="3"></a><br>
        <a href="gall3.html"><img src="gall3s.gif" width="59" height="90" border="3" vspace="5" hspace="5"></a><br>
        <a href="gall4.html"><img src="gall4s.gif" width="59" height="90" border="3" vspace="5" hspace="5"></a><br>
        <a href="gall5.html"><img src="gall5s.gif" width="60" height="90" border="3" vspace="5" hspace="5"></a><br>
        <a href="gall6.html"><img src="gall6s.gif" width="60" height="90" border="3" vspace="5" hspace="5"></a><br>
        <a href="gall7.html"><img src="gall7s.gif" width="60" height="90" border="3" vspace="5" hspace="5"></a><br>
        <a href="gall8.html"><img src="gall8s.gif" width="57" height="90" border="3" vspace="5" hspace="5"></a><br>
      </td>
    </tr>
  </table>
  <table border="0">
    <tr>
      <td><a href="..\topics.htm"><img src="ball_o.gif" width="30" height="30" border="0"></a></td>
      <td><font size="3" face="Arial, Helvetica, sans-serif"><b><font size="2">Back 
        to The Sikhism Home Page</font></b></font></td>
    </tr>
  </table>
  <p><img src="copyrigh.gif" width="243" height="70" alt="Copyright 1998 Sandeep Singh Brar. All Rights Reserved."> 
  
  <table border="0" width="300">
    <tr> 
      <td><font face="Arial, Helvetica, sans-serif" size="1">No images, photographs, 
        panoramics or movies from this web site can be reproduced, copied or used 
        in any manner without express written permission of the artist, Sandeep 
        Singh Brar.</font></td>
    </tr>
  </table>
  <p>&nbsp; </p>
</center>
<map name="gplan">
  <area shape="circle" coords="297,265,13" href="pano9.html">
  <area shape="circle" coords="43,262,14" href="pano8.html">
  <area shape="circle" coords="313,189,14" href="pano7.html">
  <area shape="circle" coords="231,165,14" href="pano6.html">
  <area shape="circle" coords="187,180,12" href="pano5.html">
  <area shape="circle" coords="163,198,14" href="pano4.html">
  <area shape="circle" coords="167,153,14" href="pano3.html">
  <area shape="circle" coords="43,109,15" href="pano2.html">
  <area shape="circle" coords="130,52,15" href="pano1.html">  
</map>
</body>
</html>
